packages/core-utils/tsconfig.json

```diff
@@ -8,6 +8,5 @@
     "lib": ["es2019", "dom"],
     "skipLibCheck": true
   },
-  "include": ["src/**/*"],
-  "exclude": ["src/__tests__/**/*"]
```

Comment:
This exclude was overriding the exclude from the extended config at the root level